engine/src/cmd/ai/aggressive.cpp

```diff
@@ -207,8 +207,8 @@ static AIEvents::ElemAttrMap * getProperScript( Unit *me, Unit *targ, bool inter
         }
         return getProperLogicOrInterruptScript( "default", fac, nam, interrupt, personalityseed );
     }
-    return getProperLogicOrInterruptScript( ROLES::getRoleEvents( me->attackPreference(),
-                                                                 targ->unitRole() ), me->faction, me->name, interrupt,
+    return getProperLogicOrInterruptScript( ROLES::getRoleEvents( me->attack_preference,
+                                                                 targ->unit_role ), me->faction, me->name, interrupt,
```

Comment:
I'm not sure about these changes - if for performance, we could make the same gain by just using inline functions while maintaining the object portability and opaque variables.

Reply:
I agree with @BenjamenMeyer here

Reply:
As noted below, I've made changes to comply, except for config values, which would come in a near PR.